wandb/sdk/launch/_project_spec.py

```diff
@@ -303,6 +313,8 @@ def build_required(self) -> bool:
         # is based on the job version index, which is immutable
         # we don't need to build the image for a job if that tag
         # already exists
```

Comment:
nit: this comment seems wildly out of place here. Can we delete it?

Reply:
removed